scripts/box/migrate_folder_language.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.mongo import database
+from framework.transactions.context import TokuTransaction
+
+from website.addons.box.model import BoxNodeSettings
+from website.app import init_app
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+def migrate(dry_run=False):
+    bns_collection = database['boxnodesettings']
+    migrated_names = {b['_id'] for b in bns_collection.find({'folder_name': 'All Files'})}
+    migrated_paths = {b['_id'] for b in bns_collection.find({'folder_path': 'All Files'})}
+    assert migrated_names == migrated_paths
```

Comment:
So if `migrated_names != migrated_paths`, more development will be needed on this migration, but you're expecting a low probability of that?